x-pack/plugins/alerting/server/routes/bulk_enable_rules.test.ts

```diff
@@ -28,7 +28,7 @@ describe('bulkEnableRulesRoute', () => {
   const bulkEnableRequest = { filter: '' };
   const bulkEnableResult = { errors: [], rules: [], total: 1, taskIdsFailedToBeEnabled: [] };
 
-  it('should enable rules with proper parameters', async () => {
+  it('should delete rules with proper parameters', async () => {
```

Comment:
Why this test name change?

Reply:
It was accident. Fixed: 27dcef8d105f49bee11222277db5a38c09d0423c